recipes/example/meta.yaml

```diff
@@ -1,82 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please don't delete the example

Reply:
@xhochy -- Sorry about that.  I've restored it now.